server/app/services/telegram_bot_service.py

```
@@ -0,0 +1,442 @@
+🇺🇸 **Luke & Jessica** - US-based agents
+
+⚠️ **Examples:**
+• `John Doe +1xxxxxxxxx`
+• `Alice Smith +91xxxxxxxxxx`
+
+Ready to try it? Send me your name and phone number now!
+        """
+        
+        await update.message.reply_text(welcome_message, parse_mode='Markdown')
+    
+    async def handle_message(self, update: Update, context: ContextTypes.DEFAULT_TYPE) -> None:
```

Ruff (0.14.7)

Unused method argument: `context`

(ARG002)